examples/auth.py

```diff
@@ -15,6 +15,18 @@ async def run():
         obo_auth_session = bdk.obo(username="username")
         logging.info(await obo_auth_session.session_token)
 
+        # update private key
+        config.setBotConfig(privateKey="/path/to/new_private_key")
```

Comment:
@symphony-soufiane I don't really understand how we can set the actual content of the key here? 

Something like: 
```python
rsa_private_key = "-----BEGIN RSA PRIVATE KEY-----...-----END RSA PRIVATE KEY-----"
config.setBotConfig(privateKeyContent=rsa_private_key)
```

Reply:
I'm not sure if we want to put the private key content in the code ? Here we give the path to the pem file like :
`config.setBotConfig(privateKey="/Users/soufiane/.symphony/rsa-localhost/privatekey.pem"`

Should it be changed to take privateKey/certificate as a string as you say ?

Reply:
Actually, the goal of this feature is to give the ability to developers to load either RSA private key or certificate content from an external system (for instance [Vault](https://www.vaultproject.io/) or [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/)). Without having to copy them on the file system as they are quite sensitive data.